bucket/mpv.net.json

```diff
@@ -0,0 +1,35 @@
+{
+    "homepage": "https://github.com/stax76/mpv.net/",
+    "description": "A modern media player for Windows that works just like mpv",
+    "license": "MIT",
+    "version": "4.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/stax76/mpv.net/releases/download/4.0/mpv.net-portable-x64-4.0.0.0.7z",
+            "hash": "0109a7f19e6ef071c8574261f987a4f5f5827c463af9f54b62d8bb8b42908f02"
+        },
+        "32bit": {
+            "url": "https://github.com/stax76/mpv.net/releases/download/4.0/mpv.net-portable-x86-4.0.0.0.7z",
+            "hash": "f9f249d17f1625c0856bfa228dc4f275b564e0eb9c22d09956b27f37e71f64ac"
+        }
+    },
+    "bin": "mpvnet.exe",
+    "shortcuts": [
+        [
+            "mpvnet.exe",
+            "mpv.net"
+        ]
+    ],
+    "persist": "portable_config",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/stax76/mpv.net/releases/download/$version/mpv.net-portable-x64-$version.0.0.7z"
```

Comment:
Autoupdate will not work for some releases:

![A](https://i.imgur.com/mSYGpZK.png)

There is need to go with something like this:

```json
"checkver": {
  "github": "repositoryURL.com",
  "regex": "download/(?<main>\\d.]+)/mpv.net-portable-x64-(?<full>[\\d.]+)\\.7z",
  "replace": "${main}" // or $main, not sure
}, autoupdate with $matchMain and $matchFull
```

Reply:
Thank you for suggestion. Full version number is used instead.